pandas/tests/arrays/test_boolean.py

```diff
@@ -251,6 +252,22 @@ def test_coerce_to_numpy_array():
         np.array(arr, dtype="bool")
 
 
+def test_to_boolean_array_from_strings():
+    result = BooleanArray._from_sequence_of_strings(["True", "False"])
+    expected = BooleanArray(np.array([True, False]), np.array([False, False]))
+
+    tm.assert_extension_array_equal(result, expected)
+
+
+def test_boolean_from_csv():
```

Comment:
this needs to go in the pandas/tests/io/parsers/test_dtypes.py

pls follow the existing patterns